activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add support for ActiveStorage expiring URLs.
```

Comment:
Could you also add an example to the docs?

Reply:
Added examples.